apps/journeys-admin/src/components/Editor/Toolbar/TitleDescriptionDialog/TitleDescriptionDialog.spec.tsx

```diff
@@ -0,0 +1,197 @@
+import { MockedProvider } from '@apollo/client/testing'
+import {
+  act,
+  fireEvent,
+  getByTestId,
+  render,
+  waitFor
+} from '@testing-library/react'
+import { SnackbarProvider } from 'notistack'
+
+import { JourneyProvider } from '@core/journeys/ui/JourneyProvider'
+
+import { defaultJourney } from '@core/journeys/ui/TemplateView/data'
+
+import { EditorProvider } from '@core/journeys/ui/EditorProvider'
+import userEvent from '@testing-library/user-event'
+import { JOURNEY_TITLE_DESCRIPTION_UPDATE, TitleDescriptionDialog } from '.'
+import { journey } from '../../../JourneyList/ActiveJourneyList/ActivePriorityList/ActiveJourneyListData'
+
+const onClose = jest.fn()
+
+describe('JourneyView/Menu/TitleDescriptionDialog', () => {
+  it('should not set journey title on close', async () => {
+    const { getByTestId, getByRole } = render(
```

Comment:
We're moving away from this way of writing tests, you can get props you need by importing screen and doing `screen.getByTestId`

```suggestion
    render(
```